infrastructure/smart-contracts/contracts/upgrades/referee/Referee9.sol

```diff
@@ -250,6 +250,18 @@ contract Referee9 is Initializable, AccessControlEnumerableUpgradeable {
         _;
     }
 
+    //TEST FUNCTION
+    function toggleAssertionChecking() public {
+        isCheckingAssertions = !isCheckingAssertions;
+        emit AssertionCheckingToggled(isCheckingAssertions);
+    }
+
+    //TEST FUNCTION
+    function setRollupAddress(address newRollupAddress) public {
+        rollupAddress = newRollupAddress;
+        emit RollupAddressChanged(newRollupAddress);
+    }
```

Comment:
Should these functions be commented out? They are required to make the MockRollup tests pass.